stories/dooboo-ui/Card.stories.tsx

```diff
@@ -18,147 +18,102 @@ const Container = styled.View`
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        title={'This is title'}
-        titleStyle={{ display: 'flex', flexDirection: 'column' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        subTitle={'This is subTitle'}
-        subTitleStyle={{ width: '100%', color: '#e4e4e4' }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-      <Card
-        titleContainerStyle={{
-          width: '100%',
-          height: '40px',
-          display: 'flex',
-          flexDirection: 'column',
-        }}
-        loading={true}>
-        <Text>{"I don't have Image"}</Text>
-      </Card>
-    </Container>
+    <ScrollView style={{ width: '100%' }}>
+      <Container>
+        <Title>{'Image & Title & SubTitle & Children'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+          title="Title"
+          subTitle="subTitle">
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'Image'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+        />
+        <Title>{'Title'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }} title="Title" />
+        <Title>{'SubTitle'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }} subTitle="subTitle" />
+        <Title>{'Title & SubTitle'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          title="Title"
+          subTitle="subTitle"
+        />
+        <Title>{'Children'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }}>
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'Title & SubTitle & Children'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          title="Title"
+          subTitle="subTitle">
+          <Text>{'children'}</Text>
+        </Card>
+      </Container>
+    </ScrollView>
   );
 }
 
-function Divider(): React.ReactElement {
+function Style(): React.ReactElement {
```

Comment:
A universal name like 'Style' may confuse others dev. Is there any specific reason to name it 'Style?

Reply:
Inline style excluded.